apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -151,7 +151,15 @@ function initializeBlocklyWrapper(blocklyInstance) {
 
   blocklyWrapper.loopHighlight = function () {}; // TODO
   blocklyWrapper.getWorkspaceCode = function () {
-    return Blockly.JavaScript.workspaceToCode(Blockly.mainBlockSpace);
+    let workspaceCode = Blockly.JavaScript.workspaceToCode(
+      Blockly.mainBlockSpace
+    );
+    if (this.getHiddenDefinitionWorkspace()) {
```

Comment:
Will there always be a `hiddenDefinitionWorkspace` for all projects?

Reply:
We think yes, but it's good to be safe here.